en/toolbar/toolbar.md

```diff
@@ -29,13 +29,13 @@ Status icons are displayed when *QGroundControl* is connected to a vehicle.
 These show the high level status of the vehicle, and can be clicked to see more detailed information. 
 
 ![](../../assets/toolbar/toolbar_status_message.jpg) **Vehicle Messages**
-<br>Click to show a dropdown of messages from the vehicle. This will change to a Yield sign if there are critical messages.
+<br>Click to show a list of messages from the vehicle. This will change to a Yield sign ![](../../assets/toolbar/toolbar_status_critical.jpg) if there are critical messages.
```

Comment:
I did not know this was call a yield sign. This is not wrong, but perhaps not widely understood. I'll modify a bit.